src/sentry/workflow_engine/handlers/condition/issue_occurrences_handler.py

```diff
@@ -3,11 +3,13 @@
 from sentry.models.group import Group
 from sentry.workflow_engine.models.data_condition import Condition
 from sentry.workflow_engine.registry import condition_handler_registry
-from sentry.workflow_engine.types import DataConditionHandler, WorkflowJob
+from sentry.workflow_engine.types import DataConditionHandler, DataConditionHandlerType, WorkflowJob
 
 
 @condition_handler_registry.register(Condition.ISSUE_OCCURRENCES)
 class IssueOccurrencesConditionHandler(DataConditionHandler[WorkflowJob]):
+    type: DataConditionHandlerType = DataConditionHandlerType.WORKFLOW_TRIGGER
```

Comment:
oops i missed this one

```suggestion
    type: DataConditionHandlerType = DataConditionHandlerType.ACTION_FILTER
```